app/src/main/java/org/mozilla/tv/firefox/FirefoxApplication.kt

```diff
@@ -50,6 +51,8 @@ open class FirefoxApplication : LocaleAwareApplication() {
         visibilityLifeCycleCallback = VisibilityLifeCycleCallback(this).also {
             registerActivityLifecycleCallbacks(it)
         }
+
+        serviceLocator.fretboardProvider.updateExperiments()
```

Comment:
Both `loadExperiments` and `updateExperiments` are called in `onCreate`, is there any reason why they're separated like this?